persist-admin-notices-dismissal.php

```diff
@@ -100,9 +100,13 @@ public static function dismiss_admin_notice() {
 		 * @return bool
 		 */
 		public static function is_admin_notice_active( $arg ) {
-			$array       = explode( '-', $arg );
-			$length      = array_pop( $array );
-			$option_name = implode( '-', $array );
+			if (strpos($arg,'-')) {
```

Comment:
Since all argument passed to `PAnD::is_admin_notice_active( 'data-notice-one-forever' )` are required to be prefixed by `data-`, this will fail and render the lib not working.

We are using the duration to get the DB option_name by popping it out of the array stack.

Dig?